### DISCUSSION TITLE
Need help moving from Version 4.8.1 to 4.13.0. Some functionalities not working.

### DISCUSSION THREAD

**Discussion by jeddtobee023** [prisma/prisma#19146] [Q&A]
We're running the code using AWS Lambda Typescript. Built using CDK.
Our DB credentials are being loaded from SSM Parameter Store.

On version 4.8.1, when running the lambda, this (pseudo) code works:
1. Prisma datasource url = env("DATABASE_URL")

2. Load the Parameter store value

3. Complete Database URL using values from Parameter store

4. process.env.DATABASE_URL = Database URL

5. run function from another file with declared import for prisma client

Now on version 4.13.0, I have this error now:
`await prisma.table1.findFirstOrThrow(
error: Environment variable not found: DATABASE_URL`

**Comment by ludralph** [comment]
Hi @jeddtobee023 👋 

Did you run `npx prisma generate` after upgrading to version 4.13.0?

**Reply by jeddtobee023** [reply]
Yes.

**Reply by ludralph** [reply]
Do you experience this in a lower version say 4.12 or 4.11?

**Reply by jeddtobee023** [reply]
Sorry. it took a while. But here are the versions I tried.

4.10.1 - not working
4.11.0 - working
4.12.0 - working
4.14.0 - not working

**Reply by ludralph** [reply]
Hi @jeddtobee023, Can you please share a minimal reproduction so we can test and verify this?

**Reply by nurul3101** [reply]
@jeddtobee023 Please let us know in case you still need any assistance.
It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
